src/components/Login.js

```diff
@@ -84,41 +35,29 @@ class Login extends Component {
              </View>
 
              {/* Render facebook login button */}
-             <Image style={styles.facebookLoginButton}
-               source={require('../resources/facebook_2x.png')}>
-               <LoginButton style= {styles.defaultButton}
-                  onLoginFinished={
-                    (error, result) => {
-                      if (error) {
-                        alert('Login has error: ' + result.error);
-                      } else if (result.isCancelled) {
-                        alert('Login is cancelled.');
-                      } else {
-                        AccessToken.getCurrentAccessToken().then(
-                          (data) => {
-
-                            // TODO: Remove this alert in production mode
-                            alert(data.accessToken.toString());
-
-                            // Move to Main if login is successful
-                            Actions.main();
-
-                          }
-                      );
-                      }
-                    }
-                }
-                onLogoutFinished={() => alert('Logout.')}/>
-            </Image>
+          <TouchableWithoutFeedback onPress={LoginUtil.signInWithFacebook}>
+              <Image style={styles.facebookLoginButton}
+               source={require('../resources/facebook_2x.png')} />
+           </TouchableWithoutFeedback>
 
             {/* Render linkedin login buttion */}
-            <TouchableHighlight onPress={this.signinWithLinkedIn}>
+            <TouchableHighlight onPress={LoginUtil.signinWithLinkedIn}>
               <Image style={styles.linkedinLoginButton}
                 source={require('../resources/Linkedin_2x.png')} />
             </TouchableHighlight>
         </View>
      );
   }
+
+  onLoginSuccess(result) {
+    Actions.main();
+  }
+
+  onLoginFail(error) {
+    if (error.code !== YodaMeta.ERR_NONE) {
+      alert(error.msg);
```

Comment:
This alert thing is only for debug purposes, right? 
Please add a TODO to remove this before we go production.